dev/fattest.simplicity/src/componenttest/topology/database/container/LibertyClientUtil.java

```diff
@@ -0,0 +1,194 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package componenttest.topology.database.container;
```

Comment:
The reason was because LibertyClient and LibertyServer don't have a great superclass between them.  If there was I would much rather have it all in one class and just cast to the subclass when needed.  But as it stands I would need to cast to the sub class for EVERYTHING. 
I could put all these methods into one class but that might be too cluttered. 

No real straight answer here.

Reply:
Ok, that is fine.  I figured it was worth the ask.